### ISSUE TITLE
Cluster Total: NaN obj/s

### ISSUE THREAD

**Issue by dvaldivia** [minio/warp#142]
Running a warp job with 4 warp clients against a 6 node MinIO with high concurrency causes the cluster total to return a `NaN`

```
warp: Benchmark data written to "warp-remote-2020-10-16[213020]-Aqi0.csv.zst"
Mixed operations.
Operation: DELETE, 0%, Concurrency: 80, Duration: 55s.
 * Throughput: 1.11 obj/s
Operation: GET, 0%, Concurrency: 80, Duration: 49s.
 * Throughput: 82.40 MiB/s, 4.32 obj/s
Operation: PUT, 0%, Concurrency: 79, Duration: 55s.
 * Throughput: 24.24 MiB/s, 1.27 obj/s
Operation: STAT, 0%, Concurrency: 79, Duration: 46s.
 * Throughput: 2.89 obj/s
Cluster Total: NaN obj/s
warp: Requesting stage cleanup start...
```

Warp Job
```
   warp-job:
    Image:      minio/warp:v0.3.21
    Port:       <none>
    Host Port:  <none>
    Args:
      mixed
      --warp-client
      warp-{0...3}.warp.ns-1.svc.cluster.local:7761
      --host
      minio.ns-1.svc.cluster.local
      --duration
      30s
      --tls
      --insecure
      --objects
      100
      --concurrent
      20
      --obj.size
      20M
    Requests:
      memory:  2Gi
```

**Comment by klauspost** [comment]
@dvaldivia attach zst file pls.

**Comment by harshavardhana** [comment]
This perhaps gets fixed with #152
